samples/matter/lock/src/lock.matter

```diff
@@ -1,9 +1,42 @@
 // This IDL was generated automatically by ZAP.
 // It is for view/code review purposes only.
 
-struct LabelStruct {
-    char_string<16> label = 0;
-    char_string<16> value = 1;
+server cluster Identify = 3 {
+  enum IdentifyEffectIdentifier : ENUM8 {
+    kBlink = 0;
+    kBreathe = 1;
+    kOkay = 2;
+    kChannelChange = 11;
+    kFinishEffect = 254;
+    kStopEffect = 255;
+  }
+
+  enum IdentifyEffectVariant : ENUM8 {
+    kDefault = 0;
+  }
+
+  enum IdentifyIdentifyType : ENUM8 {
+    kNone = 0;
+    kVisibleLight = 1;
+    kVisibleLED = 2;
+    kAudibleBeep = 3;
+    kDisplay = 4;
+    kActuator = 5;
+  }
+
+  attribute int16u identifyTime = 0;
+  readonly attribute enum8 identifyType = 1;
```

Comment:
I think we don't support identify in the application, right?

Reply:
Yes, same for window covering although both device types require it. I'm not sure if we want to implement it this late in the release, but at least I wanted to cleanup the ZAP config.

Reply:
Ok, I see, but it will be kind of bug, because our app will behave in different way than described in clusters. I understand why you did that, but maybe it would be worth to add known issue? You know just to keep this information somewhere.

Reply:
Ok, I'll try to add the support. Looks like most samples require Identify support, but they lack it.

Reply:
Done